wynnset/tapestry-wp
# 606 - Unable to Edit Activity Nodes with Questions

Closes #606 

Removes improperly formed questions from quiz on type change

Steps to test:
- Make a text node and submit
- Edit node, change to activity node
- Check "Add Question"
- Switch back to text node, submit
- Submission should work as intended
